modules/server_new/src/main/scala/observe/server/altair/AltairControllerEpics.scala

```diff
@@ -26,23 +32,18 @@ import observe.server.tcs.Gaos.ResumeCondition.OiOn
 import observe.server.tcs.Gaos.ResumeCondition.P1On
 import observe.server.tcs.Gaos.*
 import observe.server.tcs.TcsController.FocalPlaneOffset
-import observe.server.tcs.TcsEpics
+import observe.server.tcs.*
 import org.typelevel.log4cats.Logger
-import squants.Length
-import squants.Time
-import squants.space.Arcseconds
-import squants.space.Millimeters
-import squants.time.TimeConversions.*
 
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 import java.time.temporal.ChronoUnit
 import scala.annotation.unused
 
 object AltairControllerEpics {
-  final case class EpicsAltairConfig(
-    currentMatrixCoords:  (Length, Length),
-    preparedMatrixCoords: (Length, Length),
+  case class EpicsAltairConfig(
+    currentMatrixCoords:  (Quantity[Double, Millimeter], Quantity[Double, Millimeter]),
```

Comment:
`Length` seems much more legible than the `Quantity[...]`. What do you thinking of using an (opaque?) type alias?

Reply:
Not sure, at any rate it would be `LengthInMm`

Reply:
@jluhrs What do you think about using aliases for these types? They kind of confuse me but could be an option

Reply:
> Not sure, at any rate it would be `LengthInMm`

It would hide the internal representation and could have methods to extract in/construct from specific units (`toMillimeters`, `fromMillimeters`, etc.). Similar to how `Angle` behaves.